### DISCUSSION TITLE
Possible detection reasons for a R*ban.

### DISCUSSION THREAD

**Discussion by alyxme** [YimMenu/YimMenu#2134] [General]
Perhaps this topic will be useful if give some thoughts about what else can trigger RAC. 
For example, although this is not directly related, it can also be useful:
<details><summary>Screenshot</summary>
<p>

![Screenshot_20230914_221808_Discord](https://github.com/YimMenu/YimMenu/assets/62597693/a79f5e89-b226-4c8a-964a-8e8231479472)

</p>
</details>

**Comment by abuazizv** [comment]
no don't believe everything stand says its detected. ofc they will blame lua script its a paid menu...

stand got detected cuz they had the same bypass yimmenu use, stand always had shitty bypasses its better not to believe things based on nothing.

**Reply by ghost** [reply]
I am still using an outdated version of stand (the last version with free online protections) and yimmenu and I'm still not banned. If either of these menus were detected I would assume I would have joined the list of banned users. I don't see how anyone expressing their loyalty to any menu helps in figuring out why there are bans for either menu. It just clouds the air and leads to one side defending the other and we get no where.

**Comment by alyxme** [comment]
Another thoughts: is it possible for RAC to detect who KillsPlayer while using god mode?
Banned player anims/scenarios which were intended for single player only?
Timing for mission to be completed should not be as minimal as XX sec?
I dont say about obvious $ methods.
But all the thoughts about possible detection methods may helps to #makeyimmenugreatagain (c)

**Comment by ghost** [comment]
I've asked about the possibility of the injection process being detected. I only bring it up since many folks on reddit and unknowncheats mention injecting in online mode. I was under the assumption we were only supposed to inject in story mode.

**Reply by alyxme** [reply]
I used options as on this screenshot to inject. At the same time, it was these options that caused quite a few problems when updating the cache. I believed one of the community members that these are the options that allow you to hide any detection of the injected dll. However, one of the contributors here noted that detection is removed thanks to a bypass method built into the menu. 

<details><summary>Details</summary>
<p>

![injector](https://github.com/YimMenu/YimMenu/assets/62597693/06fc018d-6272-4b27-8c51-f1242ed624f6)

</p>
</details>

**Reply by ghost** [reply]
I guess I just don't understand how this bypass works to remove the detection (when the player is already in online mode) prior to injection. It sounds like putting the cart before the horse.

**Reply by alyxme** [reply]
It seems to me that the injection itself should occur before the online mode is loaded, for example during the initial boot stage, immediately after launch with no delay. As far as I know, the SC gives permission to start the online mode when the SCCLUB library is loaded, since during the ban mode there is no loading of online events on the screen.

**Comment by maybegreat48** [comment]
We know why we got detected and are working on fixing it right now. And no, this has nothing to do with Stand or lua scripts
